products/joomla.md

```diff
@@ -14,18 +14,28 @@ releaseDateColumn: true
 auto:
 -   git: https://github.com/joomla/joomla-cms.git
 
+# eol see https://developer.joomla.org/roadmap.html
 releases:
--   releaseCycle: "4"
-    eol: false
+-   releaseCycle: "5"
+    releaseDate: 2023-10-14
     support: true
+    eol: false
+    latest: "5.0.0"
+    latestReleaseDate: 2023-10-14
```

Comment:
```suggestion
    releaseDate: 2023-10-17
    support: true
    eol: false
    latest: "5.0.0"
    latestReleaseDate: 2023-10-17
```

This will be updated by the auto-configuration, but at least it will be in the git repository for future reference.